ormar/fields/many_to_many.py

```diff
@@ -124,6 +124,8 @@ def ManyToMany(  # type: ignore
 
     through_relation_name = kwargs.pop("through_relation_name", None)
     through_reverse_relation_name = kwargs.pop("through_reverse_relation_name", None)
+    is_through_relation_column_nullable = kwargs.pop("is_through_relation_column_nullable", None)
+    is_through_reverse_relation_column_nullable = kwargs.pop("is_through_reverse_relation_column_nullable", None)
```

Comment:
Can you rename it to shorter names: `through_relation_nullable` and `through_reverse_relation_nullable`